frontends/queues/resources.txt

```diff
@@ -0,0 +1,780 @@
+binheap/binheap_test.py
```

Comment:
Did we discuss that this file can just truly become a json?

Reply:
Yep, it totally can and same with `cycles.txt`, not making them jsons was pretty silly of me.

I'll tweak `cycles.sh` and `resources.sh` to do that.

Reply:
Done via [5eff826](https://github.com/calyxir/calyx/pull/2276/commits/5eff8265450afa0682caa0b60720e0a5b4ba8cbe)